test/sequential/test-repl-persistent-history.js

```diff
@@ -54,6 +54,17 @@ ActionStream.prototype.readable = true;
 const UP = { name: 'up' };
 const ENTER = { name: 'enter' };
 const CLEAR = { ctrl: true, name: 'u' };
+
+// File paths
+const fixtures = path.join(common.testDir, 'fixtures');
+const historyFixturePath = path.join(fixtures, '.node_repl_history');
+const historyPath = path.join(common.tmpDir, '.fixture_copy_repl_history');
+const historyPathFail = path.join(common.tmpDir, '.node_repl\u0000_history');
+const oldHistoryPath = path.join(fixtures, 'old-repl-history-file.json');
+const enoentHistoryPath = path.join(fixtures, 'enoent-repl-history-file.json');
+const emptyHistoryPath = path.join(fixtures, '.empty-repl-history-file');
+const defaultHistoryPath = path.join(common.tmpDir, '.node_repl_history');
+
```

Comment:
I figured it would make it easier to build all the error message strings using template strings rather than duplication `path.join` at multiple places.

In any case, I have moved this back where it was.